### DISCUSSION TITLE
特定程序定义初始状态的英文输入怎么设置才能生效？

### DISCUSSION THREAD

**Discussion by fqw000** [imfuxiao/Hamster#466] [Q&A]
特定程序定义初始状态的英文输入 app_options在weasel.custom.yaml 、squirrel.yaml配置都不起作用，hamster.yaml里应该怎么配置？还是这个功能目前不支持了？

**Comment by imfuxiao** [comment]
您好，此功能不支持。

已我目前的技术理解，iOS 的第三方键盘，拿不到键盘的宿主环境，也就无从判断是否在某个程序下运行。

如果可以得到这个信息，就可以实现这个功能。
